### DISCUSSION TITLE
Feature Request: black/white list for alert labels in the alert list visualization

### DISCUSSION THREAD

**Discussion by sdaus** [grafana/grafana#65912] [Dashboards, Panels & Visualizations]
Currently, the dashboard visualization for the [alert list](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/alert-list/) displays all labels associated with each alert, which can quickly become overwhelming when there are more than 10 labels per alert. To make it easier to focus on specific labels, a black/white list feature for the labels would be helpful.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82766. Feel free to continue the discussion around this there. Thank you!
